Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/docker.adoc

= Docker image setup
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Core Docker image setup describes how to install the Lisk SDK with Docker, the configuration & makefile & setting environment variables.

Reviewer: ```suggestion
:description: Describes all requirements and dependencies to install Lisk Core with Docker.
```
